crates/graphql/src/schema/objects/profile.rs

```diff
@@ -20,19 +21,17 @@ pub struct Profile {
 }
 
 impl From<TwitterUserProfileResponse> for TwitterProfile {
-    fn from(
-        TwitterUserProfileResponse {
-            screen_name,
-            description,
-            profile_image_url_https,
-            profile_banner_url,
-        }: TwitterUserProfileResponse,
```

Comment:
We should keep the destructing of the struct.

Reply:
ah, yes, forgot that one. Thanks